src/components/availability-panel/drag-drop-context/index.tsx

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable no-useless-escape */
+/* eslint-disable no-alert */
```

Comment:
Please refrain from disabling eslint, try to figure out what's the best practice and use it.